web/app/themes/mitlib-parent/templates/page-map-locations.php

```diff
@@ -12,13 +12,17 @@
 
 namespace Mitlib\Parent;
 
-// phpcs:disable WordPress.Security.ValidatedSanitizedInput.InputNotValidated -- need to properly receive 'v' parameter.
-$showMap = ( $_GET['v'] != '' ) && ( $_GET['v'] == 'map' ) ? 1 : 0;
-// phpcs:enable
+$v = get_query_var( 'v' );
```

Comment:
This is a fantastic example of how clear multiline conditionals are so much clearer than the single line equivalent.